website/src/components/Messages.tsx

```diff
@@ -0,0 +1,18 @@
+export interface Message {
+  text: string;
+  is_assistant: boolean;
+}
+
+const getColor = (isAssistant: boolean) => (isAssistant ? "slate-800" : "sky-900");
+
+export const Messages = ({ messages }: { messages: Message[] }) => {
```

Comment:
For a later date, this seems like a viable component for Chakra-ui style style variants.  I have an example of this in a test library I made in https://github.com/SurfaceData/surface-react-components/blob/main/src/themes/surface/components/button.ts and https://github.com/SurfaceData/surface-react-components/blob/main/src/Button/Button.tsx.

This would cut down the code and let us drop the `getColor` method.  But we can do that cleanup later.